### PR TITLE
Set page title to "Overview"; avoid custom header.

### PR DESCRIPTION
**Pull request by MarkLodato**
For index.md, manually set the page title to "Overview" rather than relying on the h1 ("Improving artifact integrity..."). This has two
effects:
1. The title in the titlebar is now "Overview", which reads better.
2. We no longer need a custom header.html template, and instead can  include the page in _config.yml.

Otherwise, there is no impact on how the page looks.

@devmoran Could you please review this change? In particular, is it OK to change the page title, or is there some SEO reason why we wanted it to be what it was before? Thanks!

### CODE REVIEW

File: docs/_layouts/home.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+---
+
+{% comment %}
+We cut out all feature of the minima "home":
```
Comment (TomHennen) [inline docs/_layouts/home.html:6]: nit: 'minimal' ?

File: docs/index.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+---
+title: Overview
```
Comment (TomHennen) [inline docs/index.md:2]: Does this just wind up rendering as 'Overview' in a tab?

Would 'SLSA Overview' be preferred?

**Review by TomHennen — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (MarkLodato) [reply docs/index.md:2]: It renders as "Overview | SLSA".

File: docs/_layouts/home.html (hunk 1 above)
Comment (MarkLodato) [reply docs/_layouts/home.html:6]: The theme name is "minima".

**Comment by MarkLodato** [comment]
Setting to draft so we don't accidentally submit without @devmoran's approval.

**Review by devmoran — APPROVED** [review]
LGTM!

**Review by TomHennen — APPROVED** [review]

**Comment by MarkLodato** [comment]
Thanks for the reviews!

**Pull request merged**